src/onegov/agency/models/person.py

```diff
@@ -53,17 +53,27 @@ def phone_es(self):
                 result.append('0' + number[-9:])
         return [r for r in result if r]
 
-    @property
-    def address_html(self):
+    @staticmethod
+    def _get_html_paragraph_with_line_breaks(prop):
         return '<p>{}</p>'.format(
```

Comment:
Use `Markup`. Since you use this function at different files, maybe better add a function in `utils`